src/kbc_modules/cc_kbc/mod.rs

```diff
@@ -48,10 +72,12 @@ impl Kbc {
 
         Kbc {
             tee: tee_type.to_string(),
+            kbs_uri,
             token: None,
             nonce: String::default(),
             tee_key: TeeKey::new().ok(),
             attester,
+            kbs_client: reqwest::Client::builder().cookie_store(true).build().ok(),
```

Comment:
1. I suggest setting user-agent (`attestation-agent-cc-kbc/VERSION`) to help with analyzing access logs in the KBS.
2. I suggest setting `.timeout(Duration::from_secs(30))` (the default is no timeout)
3. Since this becomes a very long line, maybe extract this to a `fn build_kbs_client()`.

Reply:
Updated, thanks!